src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Largest Contentful Paint
+subhead: An important, new user-centric performance metric
+authors:
+  - philipwalton
+date: 2019-08-08
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+hero_position: top
+alt: Artist painting tools
+description: |
+  This post introduces the Largest Contentful Paint API, how to use it,
+  and offers advice on how to optimize your pages for this metric
```

Comment:
This is what will appear as the teaser text on the main blog page. Please change it to match other text in structure and style. You can [see what else has been done](https://web.dev/blog). Here's a suggestion: "Use the Largest Contentful Paint API to optimize for faster page loads."

Reply:
Done.